src/04-modality-specific-files/10-microscopy.md

```diff
@@ -0,0 +1,387 @@
+| InstitutionAddress        | RECOMMENDED           | [string][]    | The address of the institution in charge of the equipment that produced the measurements.                                                                                                                 |
+| InstitutionDepartmentName | RECOMMENDED           | [string][]    | The department in the institution in charge of the equipment that produced the measurements.                                                                                                              |
+
+#### Image Acquisition
+<!---
+The following table will be generated automatically with macros after community review.
+A "manual" table is provided to facilitate the review process.
+-->
+| **Key name**               | **Requirement level** | **Data type**            | **Description**                                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| -------------------------- | --------------------- | ------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| PixelSize                  | REQUIRED              | [array][] of [numbers][] | A 2- or 3-number array of the physical size of a pixel, either`[PixelSizeX, PixelSizeY]` or `[PixelSizeX, PixelSizeY, PixelSizeZ]`, where X is the width, Y the height and Z the depth. If the file format is OME-TIFF, these values need to be consistent with `PhysicalSizeX`, `PhysicalSizeY` and `PhysicalSizeZ` OME metadata fields, after converting in `PixelSizeUnits` according to `PhysicalSizeXunit`, `PhysicalSizeYunit` and `PhysicalSizeZunit` OME fields. |
+| PixelSizeUnits             | REQUIRED              | [string][]               | Unit format of the specified PixelSize. MUST be millimeter (`"mm"`), micrometer (`"um"`) or nanometer (`"nm"`).                                                                                                                                                                                                                                                                                                                                                          |
```

Comment:
Sorry I make this comment here, since it relates to all bids specs. Aren't we promoting to use the International System for metrics? I'm seing in many places we have a place for units (not only in this bep), shouldn't this be previously converted and just keep the SI units in BIDS?

Reply:
My understanding of the Units [appendix](https://bids-specification.readthedocs.io/en/stable/99-appendices/05-units.html#unit-table) was that SI units and multiple prefixes were allowed.
As you mentionned, this is what other modalities in BIDS also do, for example `MEGCoordinatesUnits` [here](https://bids-specification.readthedocs.io/en/stable/04-modality-specific-files/02-magnetoencephalography.html#coordinate-system-json-_coordsystemjson) with a similar definition, and many others.

We chose this list because it can describe the native units of the scan system used. From a human-readability perspective, `mm`, `um` and `nm` are also more suitable than `m` for microscopy.

Reply:
Yes, I see the specs use SHOULD (not must), therefore it's only recommended. As you say units have specific fields in multiple places, so I'm completely fine for having them here too. Thank you!